workspaces/arborist/bin/reify.js

```diff
@@ -30,19 +32,23 @@ const printDiff = diff => {
   })
 }
 
-module.exports = (options, time) => {
+module.exports = async (options, time) => {
```

Comment:
This is a breaking change.

Reply:
Ah, didn't catch that this was a bin, not part of the export.  This doesn't actually change arborist itself.  All this changes is `npx -p @npmcli/arborist reify`

Reply:
Can you explain or point me in the right direction?

Reply:
it's also not a breaking change (as long as the syntax is supported in all supported node's, which i believe it is), because the function already returns a promise.